prime-router/src/main/kotlin/azure/observability/event/ReportEventService.kt

```diff
@@ -0,0 +1,498 @@
+package gov.cdc.prime.router.azure.observability.event
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped
+import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.Topic
+import gov.cdc.prime.router.azure.ActionHistory
+import gov.cdc.prime.router.azure.DatabaseAccess
+import gov.cdc.prime.router.azure.db.enums.TaskAction
+import gov.cdc.prime.router.azure.db.tables.pojos.ReportFile
+import gov.cdc.prime.router.azure.observability.context.withLoggingContext
+import gov.cdc.prime.router.report.ReportService
+import org.apache.logging.log4j.kotlin.Logging
+import org.hl7.fhir.r4.model.Bundle
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class ReportEventData(
```

Comment:
This file feels a bit overloaded. Could we move some of the constants and data classes to a separate file?

Reply:
Yea I can split it up